config/postgres_credentials_example.txt

```diff
@@ -0,0 +1 @@
+user=USER password=PASSWORD dbname=DBNAME
```

Comment:
Why you choose such config format?

Reply:
That's the format accepted by the [`PostgreSQL` driver](https://github.com/lib/pq) we are using. Is there a reason to use a different format and then convert?